sips/sip-018/sip-018-signed-structured-data.md

```diff
@@ -0,0 +1,386 @@
+# Preamble
+
+SIP Number: 018
+
+Title: Signed Structured Data
+
+Author: Marvin Janssen <https://github.com/MarvinJanssen>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Accepted
+
+Created: 28 December 2021
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Applications
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+The Signed Structured Data specification describes a standard way to
+deterministically encode and sign Structured Data. Structured Data is data that
+can be represented in human-readable format and used in applications and smart
+contracts. The aim of the standard is to produce signatures that are
+straightforward and inexpensive to verify by smart contracts.
```

Comment:
Can you add a sentence to the abstract that states that this SIP also proposes two new Clarity methods?

Reply:
Never mind, I see that this SIP doesn't itself propose new functions.